package.json

```diff
@@ -1,41 +1,39 @@
 {
-  "name": "clickhouse-query",
-  "version": "1.10.2",
-  "description": "ClickHouse Query is intuitive query builder to overcome the complexity of ClickHouse SQL syntax.",
-  "main": "dist/src/index.js",
-  "types": "dist/src/index.d.ts",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/MPLens/clickhouse-query"
-  },
-  "bugs": {
-    "url": "https://github.com/MPLens/clickhouse-query/issues"
-  },
-  "keywords": [
-    "clickhouse",
-    "sql",
-    "node"
-  ],
-  "author": "MPLens",
-  "license": "MIT",
-  "scripts": {
-    "build": "tsc",
-    "test": "jest",
-    "prepublish": "npm run build",
-    "test-publish": "npm pack",
-    "publish": "npm publish",
-    "toc": "./node_modules/.bin/markdown-toc README.md -i "
-  },
-  "dependencies": {
-    "clickhouse": "^2.6.0",
-    "winston": "^3.8.2"
-  },
-  "devDependencies": {
-    "@types/jest": "^29.2.3",
-    "jest": "^29.3.1",
-    "markdown-toc": "^1.2.0",
-    "ts-jest": "^29.0.3",
-    "ts-node": "^10.9.1",
-    "typescript": "^4.9.3"
-  }
+    "name": "clickhouse-query",
+    "version": "1.10.2",
+    "description": "ClickHouse Query is intuitive query builder to overcome the complexity of ClickHouse SQL syntax.",
+    "main": "dist/src/index.js",
+    "types": "dist/src/index.d.ts",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/MPLens/clickhouse-query"
+    },
+    "bugs": {
+        "url": "https://github.com/MPLens/clickhouse-query/issues"
+    },
+    "keywords": [
+        "clickhouse",
+        "sql",
+        "node"
+    ],
+    "author": "MPLens",
+    "license": "MIT",
+    "scripts": {
+        "build": "tsc",
+        "test": "jest",
+        "prepublish": "npm run build",
+        "test-publish": "npm pack",
+        "publish": "npm publish",
+        "toc": "./node_modules/.bin/markdown-toc README.md -i "
+    },
+    "devDependencies": {
+        "@clickhouse/client": "^0.2.9",
```

Comment:
Are you sure you mean to put `@clickhouse/client` into dev dependencies? Isn't it supposed to be in `dependencies`?